distribution/ddf-catalog/pom.xml

```diff
@@ -182,7 +182,7 @@
                         <configuration>
                             <rules>
                                 <requireFilesSize>
-                                    <maxsize>280000000</maxsize>
+                                    <maxsize>350000000</maxsize>
```

Comment:
See if this can be reduced again

Reply:
file is larger than 310000000, so reduced to 320000000